auditlog/models/rule.py

```diff
@@ -645,16 +645,26 @@ def _prepare_log_line_vals_on_create(self, log, field, new_values):
             vals["new_value_text"] = new_value_text
         return vals
 
+    def _except_active_ids(self):
+        """Extend for more field exceptions."""
```

Comment:
Can you document here what would be a good reason to refuse to show the logs for all active ids?